en/mozorg/newsletters.ftl

```diff
@@ -0,0 +1,380 @@
+newsletters-get-firefox-for-android = Get { -brand-name-firefox } for { -brand-name-android }
+
+# Name for the newsletter in Newsletter subscription page
+newsletters-download-firefox-for-ios = Download { -brand-name-firefox } for { -brand-name-ios }
+
+# Name for the newsletter in Newsletter subscription page
+newsletters-download-firefox-for-mobile = Download { -brand-name-firefox } for Mobile
+
+# Name for the newsletter in Newsletter subscription page
+newsletters-firefox-accounts-tips = { -brand-name-firefox-accounts } Tips
+
+# Description for the newsletter in Newsletter subscription page (Firefox Account Tips)
+newsletters-get-the-most-firefox-account = Get the most out of your { -brand-name-firefox-account }.
+
+# Name for the newsletter in Newsletter subscription page
+newsletters-firefox-flicks = { -brand-name-firefox } Flicks
+
+# Description for the newsletter in Newsletter subscription page (Firefox Flicks)
+newsletters-periodic-email-updates = Periodic email updates about our annual international film competition.
+
+# Name for the newsletter in Newsletter subscription page
+newsletters-firefox-ios = { -brand-name-firefox } { -brand-name-ios }
+
+# Description for the newsletter in Newsletter subscription page (Firefox iOS)
+newsletters-be-the-first-to-know = Be the first to know when { -brand-name-firefox } is available for { -brand-name-ios } devices.
+
+# Name for the newsletter in Newsletter subscription page
+newsletters-firefox-os-smartphone-owner = { -brand-name-firefox-os } smartphone owner?
+
+# Description for the newsletter in Newsletter subscription page (Firefox OS smartphone owner?)
+newsletters-dont-miss-important-news = Don’t miss important news and updates about your { -brand-name-firefox-os } device.
+
+# Name for the newsletter in Newsletter subscription page
+newsletters-firefox-os-and-you = { -brand-name-firefox-os } + You
+
+# Description for the newsletter in Newsletter subscription page (Firefox OS + You)
+newsletters-a-monthly-newsletter-and-special = A monthly newsletter and special announcements on how to get the most from your { -brand-name-firefox-os } device, including the latest features and coolest { -brand-name-firefox-marketplace } apps.
+
+# Name for the newsletter in Newsletter subscription page
+newsletters-firefox-weekly-tips = { -brand-name-firefox } Weekly Tips
+
+# Description for the newsletter in Newsletter subscription page (Firefox Weekly Tips)
+newsletters-get-a-weekly-tip = Get a weekly tip on how to super-charge your { -brand-name-firefox } experience.
+
+# Name for the newsletter in Newsletter subscription page
+newsletters-join-mozilla = Join { -brand-name-mozilla }
+
+# Name for the newsletter in Newsletter subscription page
+newsletters-maker-party = Maker Party
+
+# Description for the newsletter in Newsletter subscription page (Maker Party)
+newsletters-mozillas-largest-celebration = { -brand-name-mozilla }'s largest celebration of making and learning on the web.
```

Comment:
Is there a way to rewrite this to so it is `by Mozilla` instead of `Mozilla's` given the brand is now a placeable?

Reply:
This is a migration of some old legacy content, so I'm not going to propose rewriting it here.